### PR TITLE
Note/change phone number

### PR DESCRIPTION
**Pull request by Panoj-k**
Features

- Create change password screen
- 
image

- Create change phone number screen
- 
image
image

### CODE REVIEW

File: src/internationalization/translations/th.json [hunk 1]
```diff
@@ -195,5 +195,17 @@
     "buttonNeutral": "ไว้ทีหลัง",
     "buttonNegative": "ยกเลิก",
     "buttonPositive": "ตกลง"
+  },
+  "changePhoneNumber":{
+    "blankPhoneNumberWarning": "กรุณากรอกหมายเลขโทรศัพท์",
+    "invalidPhoneNumberWarning": "กรุณากรอกหมายเลขโทรศัพท์ที่ถูกต้อง",
+    "changePhoneNumber":"เปลี่ยนหมายเลขโทรศัพท์",
+    "enterPhoneNumber":"กรอกหมายเลขโทรศัพท์ใหม่",
+    "enterPhoneNumberDescription":"เราจะส่งรหัสยืนยันตัวตนไปให้คุณทางข้อความ อาจมีค่าบริการเพิ่มเติม",
```
Comment (chuwong35122) [inline src/internationalization/translations/th.json]: I think the translation could be clearer. The user may misunderstood.

File: src/screens/ChangePhoneNumberVerificationScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+import { SafeAreaView } from 'react-native-safe-area-context';
+import Divider from '../components/atoms/Divider';
+import OTPTimerButton from '../components/atoms/OTPTimerButton';
+import Spacer from '../components/atoms/Spacer';
+import ControlledOTPInput from '../components/molecules/ControlledOTPInput';
+import { phoneNumberVerificationCodeSchema } from '../dto/PhoneVerificationCode';
+import { useYupValidationResolver } from '../hooks/useYupValidationResolver';
+import { RootStackParamList } from '../navigation/types';
+
+const ChangePhoneNumberVerificationScreen = ({ route }) => {
```
Comment (chuwong35122) [inline src/screens/ChangePhoneNumberVerificationScreen.tsx]: add type here as
({route} :NativeStackScreenProps<RootStackParamList, 'ChangePhoneNumberVarificationScreen'>)

**Review by chuwong35122 — CHANGES_REQUESTED** [review]

File: src/internationalization/translations/en.json [hunk 3]
```diff
@@ -195,5 +195,17 @@
...
+    "blankPhoneNumberWarning": "Phone Number cannot be left blank",
+    "invalidPhoneNumberWarning": "Please input a valid phone number",
+    "changePhoneNumber":"Change Phone Number",
+    "enterPhoneNumber":"Enter Your New Phone Number",
+    "enterPhoneNumberDescription":"We’ll text you a confirmation code. Message and data rates may apply.",
+    "phoneNumber":"Phone Number",
+    "continue":"Continue",
+    "enterYourVerificationCode":"Enter Your Verification Code",
+    "verificationCodeSentTo":"Enter the verification code sent to",
+    "submit":"submit"
```
Comment (pointmekin) [inline src/internationalization/translations/en.json]: "Submit"

Comment (chuwong35122) [inline src/internationalization/translations/en.json]: submit should be capitalized? (Submit)

File: src/internationalization/translations/en.json [hunk 4]
```diff
@@ -195,5 +195,17 @@
     "buttonNeutral": "Ask Me Later",
     "buttonNegative": "Cancel",
     "buttonPositive": "OK"
+  },
+  "changePhoneNumber":{
+    "blankPhoneNumberWarning": "Phone Number cannot be left blank",
+    "invalidPhoneNumberWarning": "Please input a valid phone number",
+    "changePhoneNumber":"Change Phone Number",
+    "enterPhoneNumber":"Enter Your New Phone Number",
+    "enterPhoneNumberDescription":"We’ll text you a confirmation code. Message and data rates may apply.",
```
Comment (chuwong35122) [inline src/internationalization/translations/en.json]: A bit ambiguous

File: src/screens/ChangePhoneNumberVerificationScreen.tsx [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+              // @ts-ignore
+              !shouldDisable(errors) && watch('otp')?.length === 6
+                ? 'primary.500'
+                : 'muted.300'
+            }
+            colorScheme={'primary'}
+            variant="solid"
+            // disabled={codeValid ? true : false}
+            onPress={handleSubmit(onFormSubmit)}>
+            {t('changePhoneNumber.submit')}
```
Comment (pointmekin) [inline src/screens/ChangePhoneNumberVerificationScreen.tsx]: Consider removing commentted code

**Review by pointmekin — CHANGES_REQUESTED** [review]

File: src/internationalization/translations/th.json (hunk 1 above)
Comment (pointmekin) [reply src/internationalization/translations/th.json]: consider removing "อาจมีค่าบริการเพิ่มเติม"

File: src/internationalization/translations/en.json (hunk 4 above)
Comment (pointmekin) [inline src/internationalization/translations/en.json]: consider removing "Message and data rates may apply."

**Review by pointmekin — CHANGES_REQUESTED** [review]

**Review by pointmekin — APPROVED** [review]

**Review by chuwong35122 — APPROVED** [review]

**Pull request merged**